sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_patch.py

```diff
@@ -0,0 +1,137 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from typing import Union
+from azure.core.pipeline import PipelineRequest
+from azure.core.pipeline.policies import ( SansIOHTTPPolicy, BearerTokenCredentialPolicy, AzureKeyCredentialPolicy )
+from azure.core.credentials import ( TokenCredential, AzureKeyCredential )
+
+from ._client import TextTranslationClient as ServiceClientGenerated
+
+DEFAULT_TOKEN_SCOPE = "https://api.microsofttranslator.com/"
```

Comment:
Just checking - is this the default scope for text translation? For other cognitive services and document translation it is "https://cognitiveservices.azure.com/.default"

Reply:
Hm, maybe we will need this as well - we have our own Translator key but we also support all-in-one CS products. I will double check.

Reply:
Even for all-in-one keys, I can see https://api.microsofttranslator.com/.default is present in scopes.